cpp/modules/math.gl/core/src/core.h

```diff
@@ -379,28 +379,28 @@ class Matrix4 {
   auto operator=(Matrix4<coord> &&) -> Matrix4<coord> & = default;
 
   // Matrix Creation
-  static auto MakeUnit() -> Matrix4<coord>;
-  static auto MakeTranslation(const Vector3<coord> &offset) -> Matrix4<coord>;
-  static auto MakeScale(const Vector3<coord> &scale) -> Matrix4<coord>;
-  static auto MakeRotationX(coord angle) -> Matrix4<coord>;
-  static auto MakeRotationY(coord angle) -> Matrix4<coord>;
-  static auto MakeRotationZ(coord angle) -> Matrix4<coord>;
-  static auto MakeShearXY(coord factorX, coord factorY) -> Matrix4<coord>;
-  static auto MakeShearXZ(coord factorX, coord factorZ) -> Matrix4<coord>;
-  static auto MakeShearYX(coord factorY, coord factorZ) -> Matrix4<coord>;
-  static auto MakeProjection(coord distance) -> Matrix4<coord>;
+  static auto makeUnit() -> Matrix4<coord>;
+  static auto makeTranslation(const Vector3<coord> &offset) -> Matrix4<coord>;
+  static auto makeScale(const Vector3<coord> &scale) -> Matrix4<coord>;
+  static auto makeRotationX(coord angle) -> Matrix4<coord>;
+  static auto makeRotationY(coord angle) -> Matrix4<coord>;
+  static auto makeRotationZ(coord angle) -> Matrix4<coord>;
+  static auto makeShearXY(coord factorX, coord factorY) -> Matrix4<coord>;
+  static auto makeShearXZ(coord factorX, coord factorZ) -> Matrix4<coord>;
+  static auto makeShearYX(coord factorY, coord factorZ) -> Matrix4<coord>;
+  static auto makeProjection(coord distance) -> Matrix4<coord>;
   static auto makePerspective(coord fovy, coord aspect, coord near, coord far) -> Matrix4<coord>;
 
   auto operator()(int row, int col) -> coord & { return _m[row][col]; }
   auto operator()(int row, int col) const -> const coord { return _m[row][col]; }
 
-  auto Determinant() const -> coord;
-  auto Invert() const -> Matrix4<coord>;
-  auto Transpose() const -> Matrix4<coord>;
+  auto determinant() const -> coord;
+  auto invert() const -> Matrix4<coord>;
+  auto transpose() const -> Matrix4<coord>;
 
-  bool IsHomogenous() const;
-  auto MultiplyVector(const Vector3<coord> &) const -> Vector3<coord>;
-  auto MultiplyPoint(const Vector3<coord> &) const -> Vector3<coord>;
+  bool isHomogenous() const;
```

Comment:
I believe it should be spelled `isHomogeneous`...   https://en.wikipedia.org/wiki/Homogeneous_coordinates